### PR TITLE
Feature/[MOM-57] creating modal for settings

### PR DESCRIPTION
**Pull request by Pavlechko**
1. Created settings component
2. Created (Background, Marker, Quote, Exchange, Weather) components
3. Сreated modal with settings

### CODE REVIEW

File: src/models/Weather/weather.typse.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export type WeatherData = {
+export type Weather = {
```
Comment (cx-serhii-stanislavskyi) [inline src/models/Weather/weather.typse.ts]: You have typo in filename: weather.typSe.ts

File: src/pages/Exchange.page.tsx [hunk 2]
```diff
@@ -9,28 +9,22 @@ type Props = {
...
-        setCarrency(event.target.value as string);
-    };
+    // const handleChange = (event: SelectChangeEvent) => {
+    //     setCarrency(event.target.value as string);
+    // };
 
-    const handleChangeProvider = (event: SelectChangeEvent) => {
-        setSelectedOption(event.target.value);
-    };
+    // const handleChangeProvider = (event: SelectChangeEvent) => {
```
Comment (cx-serhii-stanislavskyi) [inline src/pages/Exchange.page.tsx]: For comments is the same as on backend.
But if need it in a future - leave it

**Review by cx-serhii-stanislavskyi — CHANGES_REQUESTED** [review]

File: src/models/Weather/weather.typse.ts (hunk 1 above)
Comment (Pavlechko) [reply src/models/Weather/weather.typse.ts]: fixed: renamed the file and corrected all its imports

File: src/pages/Exchange.page.tsx (hunk 2 above)
Comment (Pavlechko) [reply src/pages/Exchange.page.tsx]: fixed: deleted these comments

**Review by cx-serhii-stanislavskyi — APPROVED** [review]

**Pull request merged**